neo/Network/RPC/RpcServer.cs

```diff
@@ -749,5 +772,86 @@ private JObject ValidateAddress(string address)
             json["isvalid"] = scriptHash != null;
             return json;
         }
+
+        private JObject GetStateHeight()
+        {
+            var json = new JObject();
+            json["blockHeight"] = Blockchain.Singleton.Height;
+            json["stateHeight"] = Blockchain.Singleton.StateHeight;
+            return json;
+        }
+
+        private JObject GetStateRoot(JObject key)
```

Comment:
Maybe it would be better if it goes along with the `Height`, yes?

Reply:
If `key` is `Number`, return state root of the height.